src/com/facebook/buck/maven/Resolver.java

```diff
@@ -369,18 +367,18 @@ private void createBuckFiles(ImmutableSetMultimap<Path, Prebuilt> buckFilesData)
     STGroupString groups = new STGroupString("prebuilt-template", template);
 
     for (Path key : buckFilesData.keySet()) {
-      Path buckFile = key.resolve("BUCK");
-      if (Files.exists(buckFile)) {
-        Files.delete(buckFile);
-      }
+        Path buckFile = key.resolve("BUCK");
+        if (Files.exists(buckFile)) {
+          Files.delete(buckFile);
+        }
 
-      ST st = Objects.requireNonNull(groups.getInstanceOf("/prebuilts"));
-      st.add("data", buckFilesData.get(key));
-      Files.write(buckFile, st.render().getBytes(UTF_8));
+        ST st = Objects.requireNonNull(groups.getInstanceOf("/prebuilts"));
+        st.add("data", buckFilesData.get(key));
+        Files.write(buckFile, st.render().getBytes(UTF_8));
     }
   }
 
-  private Path copy(ArtifactResult result, Path destDir) throws IOException {
+  private Path copy(ArtifactResult result, Path destDir, String pkg) throws IOException {
```

Comment:
unused?

Reply:
oops you're right. Vestige from an earlier, uglier solution